docs/csharp/language-reference/builtin-types/value-types.md

```diff
@@ -13,13 +13,13 @@ ms.assetid: 471eb994-2958-49d5-a6be-19b4313f80a3
 
 *Value types* and [reference types](../keywords/reference-types.md) are the two main categories of C# types. A variable of a value type contains an instance of the type. This differs from a variable of a reference type, which contains a reference to an instance of the type. By default, on [assignment](../operators/assignment-operator.md), passing an argument to a method, or returning a method result, variable values are copied. In the case of value-type variables, the corresponding type instances are copied. The following example demonstrates that behavior:
 
-:::code language="csharp" source="~/samples/csharp/language-reference/builtin-types/ValueTypes.cs" id="ValueTypeCopied":::
+[!code-csharp[copy of values](~/samples/csharp/language-reference/builtin-types/ValueTypes.cs#ValueTypeCopied)]
```

Comment:
I'm just guessing, but the new syntax may be looking for the exact "id" unlike the old one.
We may try this on the staging site. If it didn't work, then keep the old syntax.

```suggestion
:::code language="csharp" source="~/samples/csharp/language-reference/builtin-types/ValueTypes.cs" id="SnippetValueTypeCopied":::
```

Reply:
Let's try only for this one for now :). @BillWagner does it look properly on staging?

Reply:
I’ll check tomorrow morning 
But here are some of my findings when testing this
https://github.com/dotnet/dotnet-api-docs/pull/3769#issuecomment-575528581

Reply:
Yes, the full snippet name is its ID:
https://docs.microsoft.com/en-us/contribute/how-to-write-use-markdown#named-snippet

Reply:
Also, that section says that only letters and underscore are allowed. No digits?

Reply:
I think that's probably wrong. Or it shouldn't be that way. I have a thread on this with eng. I'll point that out as well.